checkly/resource_env_var.go

```diff
@@ -0,0 +1,102 @@
+package checkly
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	checkly "github.com/checkly/checkly-go-sdk"
+)
+
+func resourceEnvVar() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEnvVarCreate,
+		Read:   resourceEnvVarRead,
+		Update: resourceEnvVarUpdate,
+		Delete: resourceEnvVarDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
```

Comment:
This schema is missing the `locked` property which we have on the API. This is a simple boolean.